packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -60,6 +60,13 @@ export interface VirtualGatewayBaseProps {
    * @default - no access logging
    */
   readonly accessLog?: AccessLog;
+
+  /**
+   * Default Configuration Virtual Node uses to communicate with Virtual Service
+   *
+   * @default - No Config
+   */
+  readonly backendDefaultsClientPolicy?: ClientPolicy;
```

Comment:
I think this name is slightly incorrect. There are multiple 'backends', but a single 'default'. So I would call this `backendsDefaultClientPolicy`.